app/src/main/java/de/westnordost/streetcomplete/quests/seating/AddSeating.kt

```diff
@@ -0,0 +1,61 @@
+package de.westnordost.streetcomplete.quests.seating
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.osmquests.Tags
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement
+
+class AddSeating : OsmFilterQuestType<Seating>() {
+    override val elementFilter = """
+        nodes, ways with amenity ~ restaurant|cafe|fast_food|ice_cream|food_court|pub|bar
+        and !outdoor_seating
```

Comment:
I'd exclude places where `takeaway=only` is tagged, because seating is implicitly already set then.

Should it also exclude places where `indoor_seating` is already tagged?

```suggestion
        and takeaway != only
        and !outdoor_seating and !indoor_seating
```

Or ask for all places where either one is missing?

```suggestion
        and takeaway != only
        and (!outdoor_seating or !indoor_seating)
```

Reply:
Yeah I'd exclude asking for takeaway only, unless you decide the response was too old and it should be re-checked.

I'd then only exclude where both outdoor_seating and indoor_seating are tagged, so your second option, because this quest is getting more complete information about both and if only one is tagged then the untagged one is undetermined.